homeassistant/components/velbus/__init__.py

```diff
@@ -62,7 +62,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Establish connection with velbus."""
     hass.data.setdefault(DOMAIN, {})
 
-    controller = Velbus(entry.data[CONF_PORT])
+    controller = Velbus(
+        entry.data[CONF_PORT], cache_dir=f"{hass.config.config_dir}/velbuscache/"
```

Comment:
Wouldn't the use if the storage folder instead of the root configuration folder make more sense in this case? As it is (assuming) not means for use consumption.

Reply:
so swiching this to:
f"{hass.config.config_dir}/.storage/velbuscache/"
?